cmake/make_versioncpp.py

```diff
@@ -105,13 +89,34 @@ def compile_output(self, template, version, branch, build_no, build_sys):
                 FreeOrion_PYTHON_VERSION="")
 
 
-if 3 != len(sys.argv):
+if 3 != len(sys.argv) and 4 != len(sys.argv):
```

Comment:
Probably it is time to start using [argparse](https://docs.python.org/3/library/argparse.html#module-argparse). But we could do it later, after merging these changes.

```suggestion
if len(sys.argv) not in (3, 4):
```

Reply:
Fixed.